README.md

```diff
@@ -347,6 +347,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [Tox](https://tox.chat/) - Distributed, secure messenger with audio and video chat capabilities. ([Source Code](https://github.com/TokTok/c-toxcore)) `GPL-3.0` `C`
 - [Tuber](https://blog.trailofbits.com/2015/12/15/self-hosted-video-chat-with-tuber/) - Peer-to-peer video chat that works. ([Source Code](https://github.com/trailofbits/tubertc)) `MIT` `Javascript`
 - [Typebot](https://typebot.io) - Typebot is a conversational app builder as an alternative to Typeform or Landbot. ([Source Code](https://github.com/baptisteArno/typebot.io)) `AGPL-3.0` `Docker`
+- [VoceChat](https://voce.chat) - Private social sphere like Slack/Discord written with Rust. Host anywhere, embed anywhere. VoceChat is for you. ([Source Code](https://github.com/privoce/vocechat-web)) `GPL-3.0` `Rust/Docker/React/Typescript`
```

Comment:
Much better, however using the following two guides: 

> https://doc.voce.chat/en-us/
https://github.com/privoce/vocechat-web#local-development
Using the local development guide due to the source code guide on docs is empty 

Please change the language to `Docker/Shell/Nodejs`